### ISSUE TITLE
Deprecation warnings on testing

### ISSUE THREAD

**Issue by arokem** [nipy/dipy#1664]
I am getting the following series of deprecation warnings while testing #1659: 

```
/Users/arokem/source/dipy/dipy/io/trackvis.py:11: DeprecationWarning: The `dipy.io.trackvis` module is deprecated as of version 0.14 of Dipy and will be removed in a future version. Please use `dipy.io.streamline` module instead
  DeprecationWarning)
/Users/arokem/source/dipy/dipy/reconst/mapmri.py:1832: DeprecationWarning: invalid escape sequence \p
  """
/Users/arokem/source/dipy/dipy/reconst/mapmri.py:1853: DeprecationWarning: invalid escape sequence \p
  """
/Users/arokem/source/dipy/dipy/reconst/mapmri.py:1877: DeprecationWarning: invalid escape sequence \p
  """
/Users/arokem/source/dipy/dipy/reconst/qtdmri.py:2116: DeprecationWarning: invalid escape sequence \D
```

Opening this issue to keep track of that.

**Comment by abhayjindal51** [comment]
I want to work on this issue.

**Comment by jhlegarreta** [comment]
@arokem Are the `trackvis` deprecation and the issues reported in the `mapmri` and `qtdmri` related?

I'd dare to say that they are not: the latter seem to be related to LaTeX syntax not being understood by Sphinx.

@abhayjindal51 I could start solving these if you do not find enough time to devote to.

**Comment by arokem** [comment]
Unrelated. I believe that the latter would be fixed either by your commit here: https://github.com/jhlegarreta/dipy/commit/8e1eb21383316c2d430164a5c4f8591fda9dfd28, or by making the entire docstring a raw string (by adding an `r` before the triple quotes).

**Comment by arokem** [comment]
And your other commit: https://github.com/jhlegarreta/dipy/commit/8df012e227196c913efc9a83e212903df815cfc4, would address the other issue, I believe.

**Comment by jhlegarreta** [comment]
So let's give a try then ! I'll open the PRs. Yes, I saw that using a raw string was another option; I just preferred to use `:math:` instead of a raw string, but if the latter are preferred, let me know.

**Comment by jhlegarreta** [comment]
Both PRs have been merged:
- The math notation issue addressed by #1701 is not present in the master CI builds.
- The `dipy.io.trackvis` deprecation warning issue addressed by #1702 is not present in the master build (there is still one such warning, but given that it stems from the `trackvis` package itself, that would not be an issue).

So I think the issue can be closed.

**Comment by arokem** [comment]
Wonderful. Thanks!
